core/lib/openapi/conversion.ts

```diff
@@ -9,6 +9,7 @@ import { clone } from '@azure-tools/linq';
 
 export async function convertOAI2toOAI3(input: DataHandle, sink: DataSink): Promise<DataHandle> {
   const converter = new Oai2ToOai3(input.originalFullPath, await input.ReadObject());
+  console.log(input.originalFullPath);
```

Comment:
Is this just for debugging or actual output?

Reply:
oops. that shouldn't be there